stbenjam/ci-test-mapping
# Initial commit

Basic structure and implementation for the storage component

- names.json has the test names dumped from sippy
- mapping.json is output from running `./ci-test-mapping --tests-file ~/names.json`

Open questions
- How to get test names?

Right now, I dump test names from Sippy (about 30K). I could also use BigQuery (`SELECT DISTINCT test_name FROM openshift-gce-devel.ci_analysis_us.junit;`) but it has a problem of pulling in 300K test names and growing because some jobs don't use deterministic test names (example: admission-control-598fc557b4-lpl79 was not OOMKilled). 

Should we only pull test names from repos like David suggested? I'm not sure about the value of that vs. just grabbing it from a the bigquery or sippy db tables of test names.
